backend/src/main/java/planit/people/preparation/Entities/Entity_User.java

```diff
@@ -58,10 +69,23 @@ public Entity_User(Long user_id, String name, String surname) {
         this.surname = surname;
     }
 
+    /**
+     * Constructor used when Entity User record needs to be created in order to save it in another table as a foreign key.
+     *
+     * @param user_id PlanIt User Id
+     */
     public Entity_User(Long user_id) {
         this.user_id = user_id;
     }
 
+    public Set<Entity_EventPreset> getEntity_EventPresets() {
```

Comment:
why not `getEntityEventPresets`?

Reply:
Auto generated getter because the field name is the_user